cibuildwheel/resources/build-platforms.toml

```diff
@@ -18,6 +18,7 @@ python_configurations = [
   { identifier = "pp39-manylinux_x86_64", version = "3.9", path_str = "/opt/python/pp39-pypy39_pp73" },
   { identifier = "pp310-manylinux_x86_64", version = "3.10", path_str = "/opt/python/pp310-pypy310_pp73" },
   { identifier = "pp311-manylinux_x86_64", version = "3.11", path_str = "/opt/python/pp311-pypy311_pp73" },
+  { identifier = "gp242-manylinux_x86_64", version = "3.11", path_str = "/opt/python/graalpy311-graalpy242_311_native" },
```

Comment:
Wheels have to be made for each minor version too? I guess this means we can't drop all the workarounds until we drop `gp242`?

Reply:
We are doing patch releases, I missed the merge window for a patch release that goes out next week, but I'm confident I can get the required fixes in for our patch release in June, at which time we can drop this.

Versioning is a bit confusing right now, we used to do calendar versioning with two releases per year, but we are switching to align with the OpenJDK release schedule and will then have 2 releases per year. At that point every 4th release will be a long-term support release, releases in between will only receive support for 6 months. The upcoming release 25 (due out in Sep) will be an LTS, so we could then stick to that for 2 years and also have 26/27/28 each for 6 months until we go to 29 at which point 25 is out of support.

Reply:
So in terms of ABI stability, a sample wheel I just produced was given the filename
`spam-0.1.0-graalpy311-graalpy242_311_native-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_28_x86_64.whl`. So it does look like there's only ABI stability within each minor release of GraalPy (i.e. across patch releases), is that right? 

(I just want to check the logic behind this build identifier, as most have the python version at the start, not the interpreter version. But if the ABI stability is tied to the GraalPy minor version, this makes sense to me.)

Reply:
I don't think we've got a plan for how to handle this if PyPy ever has a new ABI other than 7.3?

I'd assume GraalPy would hopefully bump the Python version once a year, but that's still two releases per Python version, so it seems unavoidable.

Reply:
Triggered by this discussion, we are changing the way handle ABI changes and will align with CPython in that we only break our ABI once a year, in the GraalPy release that happens around the same time of the CPython release in the latter half of the year. So our spring release will keep the ABI stable and then adopting new GraalPy ABI will happen in the same timeframe were cibuildwheels and other projects update to the new CPython version, too. Would that be ok?